pype/nukestudio/lib.py

```diff
@@ -350,17 +354,20 @@ def CreateNukeWorkfile(nodes=None,
     # create root node and save all metadata
     root_node = hiero.core.nuke.RootNode()
 
-    root_path = os.environ["AVALON_PROJECTS"]
+    if not os.getenv("PYPE_ROOT_WORK"):
+        set_multiroot_env()
```

Comment:
Not defined function. These environments are already set and if anyone wouldn't have set root["work"] then can't publish. REMOVE PLEASE